src/game-board/game-board.html

```diff
@@ -90,8 +90,7 @@
       */
       _cardDropped(e){
 
-        if(this.movingStack.stack.cards[this.movingStack.stack.cards.length-1].rank === this.movingCard.rank
-          && this.movingStack.stack.cards[this.movingStack.stack.cards.length-1].suit === this.movingCard.suit){
+        if(this._canCardMoveHere(e.detail.stack.cards[e.detail.stack.cards.length -1].rank, this.movingCard.rank)){
```

Comment:
I am fairly certain, as e.detail.stack.cards is our destination stack, and in bakers dozen you are not allowed to fill an empty stack, the only way the destination stack would be empty would be if we were trying to put it somewhere empty, and this is not allowed anyway.